src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Lottery extends RandoM {
```

Comment:
Extending the `RandoM` class here might not be the best design choice. This implies a 'is-a' relationship, which doesn't seem to fit the context. Instead, consider using composition over inheritance. You can create an instance of `RandoM` inside the `Lottery` class and use its methods.